### PR TITLE
Update/fix e2e tests to work with Golang operator 

### PR DESCRIPTION
**Pull request by eemcmullan**
#214

### CODE REVIEW

**Comment by kaovilai** [comment]
Hi Emily. Rebase this to openshift/master and we can retest operator-e2e

File: tests/e2e/install_suite_test.go [hunk 1]
```diff
@@ -8,68 +8,85 @@ import (
...
 })
 
 var _ = AfterSuite(func() {
 	// Check Velero is deleted
-	Eventually(isVeleroDeleted(namespace, testSuiteInstanceName), time.Minute*2, time.Second*5).Should(BeTrue())
+	Eventually(vel.IsDeleted(), time.Minute*2, time.Second*5).Should(BeTrue())
 
 	// Check Restic daemonSet is deleted
-	Eventually(isResticDaemonsetDeleted(namespace, testSuiteInstanceName, resticName), time.Minute*2, time.Second*5).Should(BeTrue())
+	//Eventually(isResticDaemonsetDeleted(namespace, testSuiteInstanceName, resticName), time.Minute*2, time.Second*5).Should(BeTrue())
```
Comment (dymurray) [inline tests/e2e/install_suite_test.go:36]: Can we remove this?

File: tests/e2e/install_suite_test.go [hunk 2]
```diff
@@ -8,68 +8,85 @@ import (
...
 
 	// Check secret is deleted
-	Eventually(isCredentialsSecretDeleted(namespace, credSecretRef), time.Minute*2, time.Second*5).Should(BeTrue())
+	//Eventually(isCredentialsSecretDeleted(namespace, credSecretRef), time.Minute*2, time.Second*5).Should(BeTrue())
```
Comment (dymurray) [inline tests/e2e/install_suite_test.go:39]: Remove?

**Comment by eemcmullan** [comment]
@dymurray I removed those lines.

**Comment by dymurray** [comment]
Thank you @eemcmullan , I had to make some changes with go module deps so I added your commits here: https://github.com/openshift/oadp-operator/pull/236/files. Let's close this one so we can catch these changes. Your commits are still in the other one and will cherry-pick the last commit you have

**Pull request closed without merging**